sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,33 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server
+
+  async getGlobalConf(gatewayId, baseURL, token) {
```

Comment:
```suggestion
  async getGlobalConf(gatewayId, baseURL) {
```

Instead, pass the token on to the gateway service from `sdk/js/src/index.js`.